shm_wrapper/shm_wrapper.c

```diff
@@ -775,7 +775,11 @@ int gamepad_read(uint32_t* pressed_buttons, float joystick_vals[4]) {
 
     // if no gamepad connected, then release rd_sem and return
     if (rd_shm_ptr->fields[GAMEPAD] == DISCONNECTED) {
-        log_printf(ERROR, "tried to read, but no gamepad connected");
+        *pressed_buttons = 0;
+        for (int i = 0; i < 4; i++) {
+            joystick_vals[i] = 0;
+        }
+        log_printf(DEBUG, "tried to read, but no gamepad connected");
```

Comment:
It is an error on purpose, so the student knows they are running faulty code. As of now, this is only called in the student API. Right now, it looks like it will print 2 error statements, one from the logger and one from a Python exception. Is that why you wanted to make it debug?

Reply:
I can switch it back to `ERROR`. How does the student know that their code is faulty if this log appears? This means either that 1) they don't have a gamepad connected in `TELEOP` or 2) they're trying to access gamepad data in `AUTO`?

The main reason I changed it was because it was giving me issues while working on the shm UI. The UI polls for the gamepad state multiple times per second. I tested each component of the UI individually, many of which didn't need a gamepad connected. As a result, my logs were being overwhelmed with the `ERROR` message, so I couldn't use other logs to debug my work. My temporary workaround was to simply comment out the log.

If any user wants to use the shm UI, they'll want to hide this log message so that they can see other logs. What do you recommend I should do here?